sungc/deglint.py

```diff
@@ -459,14 +488,14 @@ def nir_subtraction(self, vis_band_ids, nir_band_id, odir=None, waterVal=5):
             The path where the deglinted geotiff bands are saved.
 
             if None then:
-            odir = pjoin(self.group_path, "DEGLINT", "GAO")
+            odir = pjoin(self.group_path, "DEGLINT", "MINUS_NIR")
```

Comment:
Just checking that this should be "MINUS_NIR" and not "GAO"

Reply:
I changed the folder name from GAO to MINUS_NIR to better represent what the algorithm does